client/public/clientJS/stripe.js

```diff
@@ -53,6 +56,7 @@ form.addEventListener('submit', function (event) {
         .catch((error) => {
           console.error('Error:', error);
           showToast('An error has occured while processing payment');
+          loading(false);
```

Comment:
do we need to add aother loading(false) call in `payWithCard`?

Reply:
Yes, if it fails on our backend we need it to re enable the pay button again